### PR TITLE
Output synchronized image model with video

### PR DESCRIPTION
**Pull request by mostajabi**
This commit will write synchronized image model with output video. This will provide information required to align Shannon video output with lidar and camera data.

### CODE REVIEW

File: python/viewer/image_viewer.py [hunk 1]
```diff
@@ -57,12 +61,14 @@ def main():
     for radar_name in args.radar_name:
         output_path = None
         if args.output_dir is not None:
-            output_path = join(args.output_dir, radar_name+".mp4")
+            video_output_path = join(args.output_dir, radar_name+".mp4")
+            image_model_output_path = join(args.output_dir, radar_name+".pbs")
```
Comment (jimmy-zendar) [inline python/viewer/image_viewer.py]: we should give it a more descriptive name, such as  radar_name + "_image_models.pbs".

File: python/viewer/radar_data_streamer.py [hunk 2]
```diff
@@ -29,10 +30,14 @@ def __init__(self, data_stream_path, proto_type, python_type, header_size=8):
...
-        self.data_streams = None
+        self.mode = mode
+        self.data_stream = None
 
     def __enter__(self):
-        self.data_stream = open(self.data_stream_path, "rb")
+        self.data_stream = open(self.data_stream_path, self.mode)
+        if self.mode == 'wb':
+            # write 8 bytes header
+            self.data_stream.write(b"\x41\x48\x41\x56\x00\x00\x00\x00")
```
Comment (jimmy-zendar) [inline python/viewer/radar_data_streamer.py:40]: does this string of bytes actually mean anything? Is that better than just pack it with all 0s?

**Review by jimmy-zendar — APPROVED** [review]
Looks good to me.

File: python/viewer/radar_data_streamer.py (hunk 2 above)
Comment (mostajabi) [reply python/viewer/radar_data_streamer.py:40]: For consistency I used what we use in` io.py` in the tracker pipeline.

**Comment by mostajabi** [comment]
Synchronized image models with video frames.

**Pull request merged**